content/Hardware Support/Portenta Family/FAQ-Arduino-Opta.md

```diff
@@ -11,6 +11,15 @@ The frequently asked questions covered in this article are focused on providing
 
     Opta® supports a variety of programming languages, including the Arduino programming language and standard IEC-61131-3 PLC programming languages such as Ladder Diagram (LD), Sequential Function Chart (SFC), Function Block Diagram (FBD), Structured Text (ST), and Instruction List (IL). This flexibility makes it suitable for automation engineers with different programming preferences.
 
+- **Where can I get documentation on IEC 61131-3 programming languages?**
+
+    To program Opta® using IEC 61131-3 programming languages, it is necessary to download a software, called [PLC IDE](https://www.arduino.cc/en/software#arduino-plc-ide). Tutorials and documentation can be found on the [Arduino PLC IDE Documentation page](https://docs.arduino.cc/software/plc-ide/) and an online manual is integrated in the software; it can be accessed from top menu bar, clicking on “Help”-->”Index”. This manual can be downloaded and stored locally or even printed as a reference and contains information on the programming environment overview and how to use it, projects and project elements management, programming languages use and functions as well as debugging tools info.
+
+- **Is is possible to migrate PLC code built on different software environments from other vendors into PLC IDE to make it work with Opta?**
+
+    The IEC 61131-3 programming languages are standard and program organizational units (POUs) can be ported with a reasonable effort, as long as the original code does not imply proprietary parts (e.g. libraries) for which the source is not shared. Usually it is a matter of time dedicated to the copy, after managing variables creation and possible vendor-dependent function blocks re-writing.
```

Comment:
```suggestion
    The IEC 61131-3 programming languages are standardized, and program organizational units (POUs) can be ported with reasonable effort, as long as the original code does not include proprietary parts (e.g., libraries) for which the source is not shared. Usually, it is a matter of dedicating time to copying the code, after managing variable creation and possibly rewriting vendor-dependent function blocks. For ST language, this process is faster because it’s text-based, and an editor copy usually works. For graphical languages, it takes more time as the POUs must be rebuilt.
```